.github/workflows/native-cron-build.yml

```diff
@@ -0,0 +1,64 @@
+name: "Quarkus Cron Native image"
+on:
+  schedule:
+    - cron: '0 2 * * *'
+  repository_dispatch:
+
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        java: [ 8, 11 ]
+    name: build-and-testing
+    steps:
+
+      - name: Stop MySQL
+        run: sudo systemctl stop mysql
+
+      - name: Pull docker image
+        run: docker pull quay.io/quarkus/ubi-quarkus-native-image:19.3.1-java${{ matrix.java }}
+
+      - name: Pull docker image
+        run: docker pull quay.io/quarkus/ubi-quarkus-native-image:19.3.1-java${{ matrix.java }}
+
+      - name: Set up Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java }}
+
+      - name: Checkout Quarkus
+        uses: actions/checkout@v2
+        with:
+          repository: quarkusio/quarkus
+          ref: master
+
+      - uses: actions/cache@v1
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+
+      - name: Build Quarkus
+        run: mvn -B install -DskipTests -DskipITs -Dno-format
+
+      - name: Run integration tests in native
+        run: mvn -B  --settings azure-mvn-settings.xml verify -f integration-tests/pom.xml -DskipTests  -Dno-format -Ddocker -Dnative -Dquarkus.native.container-build=true -Dquarkus.native.builder-image=quay.io/quarkus/ubi-quarkus-native-image:19.3.1-java${{ matrix.java }} -Dtest-postgresql -Dtest-elasticsearch -Dtest-mysql -Dtest-dynamodb -Dtest-vault -Dtest-neo4j
+
+      - name: Report
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_API_TOKEN }}
+          STATUS: ${{ job.status }}
+        run: |
+          echo "The report step got status: ${STATUS}"
+          sudo apt-get update -o Dir::Etc::sourcelist="sources.list" \
+            -o Dir::Etc::sourceparts="-" -o APT::Get::List-Cleanup="0"
+          sudo apt-get install -y gnupg2 gnupg-agent
+          echo Installing SDKMAN
+          curl -s "https://get.sdkman.io" | bash
+          source ~/.sdkman/bin/sdkman-init.sh && \
+          sdk install kotlin 1.3.61 && \
+          sdk install kscript 2.9.0
+          kscript .github/NativeBuildReport.kts ${GITHUB_TOKEN} ${STATUS}
```

Comment:
We would need the Java version in the report, wouldn't we?

Reply:
Very good point. We actually will probably need two issues. One for Java 8 and one for Java 11 since we need to open/closing of the issues to happen on each specific one.

WDYT?

Reply:
The point should be addressed now. It's not the most elegant syntax in the world, but it seems like github actions doesn't give us many options